packages/botonic-nlp/tests/parser/data-augmenter.test.ts

```diff
@@ -0,0 +1,23 @@
+import { DataAugmenter } from '../../src/parser/data-augmenter'
+
+describe('Data augmentation', () => {
+  test('Generating variations of the given samples', () => {
+    expect(
+      DataAugmenter.augment(
+        [
+          'Where can I buy this [material] [product]?',
+          "I don't like this [material] [jacket](product)",
+        ],
+        { product: ['jacket', 'hoodie'], material: ['leather', 'cotton'] },
+        ['O', 'product']
+      )
```

Comment:
This applies to all tests:
1. Cache first the variable you are going to test (your `Subject under test`)
2. `sut = DataAugmenter.augment(...)`
3. èxpect(sut).toEqual(value) (or cache the expected value in `expected` and just do expect(sut).toEqual(expected)